lib/ui/window.dart

```diff
@@ -294,3 +311,21 @@ class Window {
 /// core scheduler API, the input event callback, the graphics drawing API, and
 /// other such core services.
 final Window window = new Window._();
+
+/// Platform message for queueing.
+class _PlatformMessage {
+  final String name;
+  final ByteData data;
+  final PlatformMessageResponseCallback responseCallback;
+
+  _PlatformMessage(this.name, this.data, this.responseCallback);
+}
+
+/// Queue of unhandled [_PlatformMessage]s.
+final List<_PlatformMessage> _messageQueue = <_PlatformMessage>[];
+
+/// Queueing platform message handler.
+final PlatformMessageCallback _queueingPlatformMessageCallback =
+  (String name, ByteData data, PlatformMessageResponseCallback responseCallback) {
+    _messageQueue.add(new _PlatformMessage(name, data, responseCallback));
```

Comment:
It looks like this buffer will grow without bound if no one ever registers an onPlatformMessage callback.